### PR TITLE
BETA: Register heyavi.is-a.dev

### PR DESCRIPTION
**Pull request by akhileshrangani4**
Added `heyavi.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/akhileshh.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Akhileshrangani4",
+        "email": "akhileshrangani4@gmail.com"
+    },
+    "record": {
+        "TXT": "google-site-verification=mxvu1et47munvjqueyoocxtlg8u1ikizyueuxntqtka"
```
Comment (Stef-00012) [inline domains/akhileshh.json:7]: Google site verification through TXT records is not supported, use the HTTPS method

File: domains/akhilesh.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Akhileshrangani4",
+        "email": "akhileshrangani4@gmail.com"
+    },
+    "record": {
+        "TXT": "google-site-verification=jr4pbfhu2ujqjmfocyc-ukbxokjugvn8oit17d-qhia"
```
Comment (Stef-00012) [inline domains/akhilesh.json:7]: Google site verification through TXT records is not supported, use the HTTPS method

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by Akhileshrangani4** [comment]
prevent

**Comment by DEV-DIBSTER** [comment]
Closing for inactivity.

**Pull request closed without merging**